Directory.Build.props

```diff
@@ -87,6 +87,9 @@
     <PackageVersion>$(DotNetVersionBand)-dev</PackageVersion>
     <PackageOutputPath>$(MSBuildThisFileDirectory)artifacts</PackageOutputPath>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Nullable" Version="1.3.0" PrivateAssets="all" />
+  </ItemGroup>
```

Comment:
This is causing a whole bunch of extra errors. I wonder if either we skip the nullable attributes on netstandard or we only include this on netstandard?

Reply:
What were the errors? I couldn't figure them out from the CI

Added condition to only include it for netstandard2.0